### ISSUE TITLE
Poke::API::Errors::LoginFailure

### ISSUE THREAD

**Issue by G-E-N-E** [dphuang2/PoGoBag#104]
Noticed on the official site when it was up that the refresh was not actually updating the bag.
This continued when I setup a local instance and here is the error.
`[2017-01-03T00:03:03-05:00]: DEBUG > Poke::API::Auth::GOOGLE   --: [>] Fetching Google access token
Completed 500 Internal Server Error in 1895ms (ActiveRecord: 1.7ms)
  
Poke::API::Errors::LoginFailure (Unable to login to GOOGLE => [Unable to login to Google, could not find => Token in {"Error"=>"BadAuthentication"} @ $INSTALL_PATH/lib/poke-api/auth/google.rb:71:in `perform_request']):
`
Initial sign in with google seems to work, and update if I re-signin.

Known Issue that you have to get a new authorization key every time you want to update your bag?

**Comment by Gladiator10864** [comment]
Yeah, I've been experiencing the same issue. Though as you said, you can still get around this by getting a new token. Not sure yet why it fails the first time.
